### PR TITLE
Bump doorkeeper version to 2.1.4

### PR DESCRIPTION
**Pull request by takashi**
If there is any reason to keep doorkeeper's version with 1.4.x, please teach me :bow:

### CODE REVIEW

File: garage.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", '>= 4.0.0'
-  s.add_dependency "doorkeeper", ">= 0.6.7", "< 1.5.0"
+  s.add_dependency "doorkeeper", "~> 2.1.4"
```
Comment (taiki45) [inline garage.gemspec]: It seems [doorkeeper gem has `doorkeeper_authorize!` from v2.0.0](https://github.com/doorkeeper-gem/doorkeeper/blob/master/NEWS.md#200---2014-12-16). Are there any reason to define dependency as `~> 2.1.4`? I think defining  `>= 2.0.0` is enoght for the incompatible `doorkeeper_for` change.

**Comment by taiki45** [comment]
We welcome to support doorkeeper v2.0.0 and later and to drop support for v1.4.2 or earlier.

File: garage.gemspec (hunk 1 above)
Comment (takashi) [inline garage.gemspec]: @taiki45 you are right, Il' changed this to `>= 2.0.0` and squashed commit

**Comment by takashi** [comment]
@taiki45 thank you for reviewing. I fixed version of dependency in gemspec

**Comment by taiki45** [comment]
Thank you for sending this patch.

I merged this patch with squashing and editing some commit message by 2a127221d0f4fcac41825bcf0050582765a4d5f6.

**Comment by takashi** [comment]
@takai thank you for merging!

**Pull request closed without merging**